src/components/Popover/index.jsx

```diff
@@ -1,300 +1,66 @@
-
-  /**
-   * JSX - Content to place in the positioned popover container. The popover container
-   * does not provide any default styling; content should be styled with background,
-   * borders, and shadows as appropriate.
-   */
-  children: PropTypes.oneOfType([PropTypes.node, PropTypes.element]).isRequired,
-
-  /**
-   * What mode of interaction triggers the active/inactive state of the popover.
-   *
-   * `hover` - popover opens on trigger hover/focus
-   *
-   * `click` - popover opens on trigger click
-   *
-   * `controlled` - enables "fully controlled" mode in which the popover is only active
-   * when the `isOpen` prop is set to `true`. Use `onUserDismiss` to handle dismissal
-   * events from user interaction.
-   */
-  interactionMode: PropTypes.oneOf(VALID_INTERACTION_MODES),
-
-  /** Controlls active state of popover when in fully controlled interaction mode */
+  /** the UI display */
+  theme: PropTypes.oneOf(['default', 'unthemed']),
+  children: PropTypes.node,
+  /** Element that popover is positioned on */
+  triggerElement: PropTypes.node,
+  /** Position preference of the popover */
+  placement: PropTypes.oneOf(['auto', 'top', 'right', 'bottom', 'bottom-start', 'left']),
+  /** Controls the popover */
   isOpen: PropTypes.bool,
-
-  /**
-   * Callback fired when user takes an action to dismiss the popover.
-   * (e.g. ESC press, clickikng outside, etc.)
-   * Useful for updating `isOpen` in controlled mode.
-   * Function is invoked with the DOM event as the argument.
-   */
+  /** calls this callback when user clicks outside popover or presses esc key */
   onUserDismiss: PropTypes.func,
-
-  /** disables portaling the popover to `document.body` */
-  disablePortal: PropTypes.bool,
-
-  /**
-   * Vertical position preference of popover content.
-   * `top` for example, will place the popover content above the trigger.
-   */
-  position: PropTypes.oneOf(VALID_POSITIONS),
-
-  /** Horizontal alignment preference of popover content relative to trigger */
-  alignment: PropTypes.oneOf(VALID_ALIGNMENTS),
-
-  /** Offset distance from trigger. */
-  distance: PropTypes.number,
-
-  /** Delay in milliseconds for popover to trigger (hover mode only) */
-  delay: PropTypes.number,
-
-  /** Name of transition for popover content */
-  transitionName: PropTypes.oneOf(['GrowFast']),
-
-  /** Callback called when popover opens */
-  onOpen: PropTypes.func,
-
-  /** Callback called when popover closes */
-  onClose: PropTypes.func,
-
-  /** When popover opens, this element should freeze scrolling. When the popover closes, the scrollability of this element should resume. */
-  disableScrollRef: validRef,
-
-  /** When the Popover detects scroll events from this ref, the popover should close. */
-  closeOnScrollRef: validRef,
-
-  /** Ref to forward to the content container of the popover */
-  ref: validRef,
+  /** What mode of interaction triggers the popover */
+  trigger: PropTypes.oneOf(['click', 'hover']),
```

Comment:
In the current implementation, `trigger` refers to the trigger element. I like the `triggerElement` prop name for the tippy popover because it's explicit.

How would you feel about making this name explicit as well (`triggerType`)?